src/main/java/org/broadinstitute/consent/http/db/mapper/DarCollectionSummaryReducer.java

```diff
@@ -31,6 +31,10 @@ public void accumulate(Map<Integer, DarCollectionSummary> map, RowView rowView)
         summary.addDatasetId(datasetId);
       }
 
+      if (hasColumn(rowView, "dac_name", String.class)) {
+        summary.addDacName(rowView.getColumn("dac_name", String.class));
+      }
```

Comment:
In testing, I saw this:
```
    "dacNames": [
      "DAC 0001",
      "DAC 0002",
      null,
      "Demo Dac New"
    ],
```
and this
```
    "dacNames": [
      null
    ],
```
We should probably do a little filtering here to ensure we're not adding null values.